.github/ISSUE_TEMPLATE/Feature_request.yml

```diff
@@ -0,0 +1,20 @@
+name: Feature request
+description: Report a bug with the WordPress block editor or Gutenberg plugin
+title: 'Feature request: '
+body:
+    - type: textarea
+      attributes:
+          label: What problem does this address?
+          description: Please describe if this feature or enhancement is related to a current problem or pain point.
+          placeholder: |
+              I'm always frustrated when '...'. It is currently difficult to '...'.
+      validations:
+          required: true
+    - type: textarea
+      attributes:
+          label: What is your proposed solution?
+          description: Please outline the feature or enhancement that you want and how it addresses any problem identified above.
+          placeholder: |
+              It would be great if '...' was posible.
```

Comment:
In that example, I'm picturing `add a new control in a block` as the proposed solution rather than the problem at hand. The problem would be they want to do something but they don't have the right tools, hence asking for the new control (without knowing the technical implications of that proposal). Another example:
- Problem: I can't add styles to my post title.
- Proposed solution: Allow bold, underline in the post title. (Some would even propose "Enable rich text in post title").

In this scenario, the implementation of that solution to their problem would mean changing the way the post title is stored to allow rich text, but I wouldn't expect most feature requests to go that deep into the "solution".

I would love to hear more opinions, as I am sure this has already been debated when the original template this form replicates was created. In any case, the fact that we are discussing this is a hint that it may not be a clear enough format for folks opening issues 😅